pkgs/by-name/hu/humanshell/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "humanshell";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "juanparati";
+    repo = "humanshell";
+    rev = "v${version}";
+    sha256 = "sha256-u2aCZVfh5ag6/AN6x9ewVxl21f9UwJ6rudtwK9VRQ4c=";
+  };
+
+  cargoHash = "sha256-4oyP8oXscmbiZRAK22RwZ5k6vgrCSWAq58SVf3m9zzg=";
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242